ui/app/src/state/reducers/uiConfig.ts

```diff
@@ -64,6 +64,7 @@ const initialState: GlobalState['uiConfig'] = {
     bulkPublishCommentRequired: false,
     publishByCommitCommentRequired: false,
     publishCommentRequired: false,
+    publishEverythingCommentRequired: false,
```

Comment:
Do we need to update the blueprints? (the samples and/or default configs)
cc @sumerjabri

Reply:
@jvega190 you can perform bp updates under same ticket